plugin/README.md

```diff
@@ -0,0 +1,28 @@
+# Verifiable Credential Service Worker Plugin for ReSpec
+
+But with support for v2, and no JSON-LD processing.
+
+# Usage
+
+To use this extension, add the `respec-plugins` directory to your spec,
+then configure respect to use the worker to post process like so:
+
+```html
+<head>
+  <title>Respec Service Worker Plugin Test</title>
```

Comment:
```suggestion
  <title>ReSpec Service Worker Plugin Test</title>
```

Reply:
thanks @TallTed similar to my response to David, I will incorporate your changes in a follow-up PR.